test/helper/sample-schema/articles/articles.go

```diff
@@ -0,0 +1,95 @@
+package articles
+
+import (
+	"github.com/go-openapi/strfmt"
+	"github.com/google/uuid"
+	"github.com/semi-technologies/weaviate/entities/models"
+)
+
+func ArticlesClass() *models.Class {
```

Comment:
I love the new schema example! It's cool that we gather those so that we can re-use them in tests